crates/aptos/src/common/types.rs

```diff
@@ -1288,6 +1292,11 @@ pub struct TransactionOptions {
     pub(crate) gas_options: GasOptions,
     #[clap(flatten)]
     pub(crate) prompt_options: PromptOptions,
+
+    /// If this option is set, simulate the transaction locally using the debugger and generate
+    /// flamegraphs that reflect the gas usage.
```

Comment:
I would add a "short" description that will show up with `aptos -h` instead of `aptos --help`
```suggestion
    /// Profile the transaction's gas cost
    ///
    /// If this option is set, simulate the transaction locally using the debugger and generate
    /// flamegraphs that reflect the gas usage.
```